app/views/course/condition/_conditions.html.slim

```diff
@@ -0,0 +1,25 @@
+- conditional_name = conditional.class.name.demodulize.singularize.downcase
+
+h2
+  span = t('course.condition.header')
+
+div.dropdown.pull-right
+  button.btn.btn-primary.dropdown-toggle#new-condition-dropdown-btn [data-toggle="dropdown"
+    aria-haspopup="true"]
+    span = t('course.condition.new.header')
+  ul.dropdown-menu.dropdown-menu-right aria-labelledby="new-condition-dropdown-btn"
+    - Course::Condition::NAMES.each do |condition_name|
+      li= link_to t("course.condition.#{condition_name.pluralize}.new.header"),
```

Comment:
Google. I can't possibly be giving you answers to every single question. `I18n.t('helpers')` in an irb.